Sources/PathControl/PopUpPathControl.swift

```diff
@@ -25,14 +25,18 @@ public struct PopUpPathControl: NSViewRepresentable {
     /// Creates a pop-up path control with default contents.
     ///
     /// - Parameter url: A binding to property that defines the currently-selected url.
-    public init(url: Binding<URL?>) {
+    public init(url: Binding<URL?>, fileChooser: Bool = false) {
```

Comment:
I'd prefer to avoid introducing behavior changes to existing users of the package. To do that this should default to `true` preserving old behavior.

```suggestion
    public init(url: Binding<URL?>, fileChooser: Bool = true) {
```